testing/libtss2: new aport

whooo opened the pull request.

Cogitri commented on testing/libtss2/APKBUILD:
```diff
@@ -0,0 +1,138 @@
+
+build() {
+    cd "$builddir"
+    ./configure CFLAGS="-fno-inline" \
```
This still applies

Cogitri commented on testing/libtss2/APKBUILD:
```diff
@@ -0,0 +1,90 @@
+    pkgdesc="TPM 2.0 (static libraries)"
+    mkdir -p "$subpkgdir/usr/lib"
+    mv "$pkgdir"/usr/lib/*.a "$subpkgdir/usr/lib"
+}
```
abuild should do this automatically for you if you add $pkgname-static to subpackages

Cogitri commented:
Please only indent with tabs. I'd recommend you to install atools for apkbuild-lint which can automatically find such errors.

Also: `testing/libtss2/APKBUILD:15:unnecesary usage of braces: ${pkgver}`

asac commented:
Hi, if we want the more fine grained split that this package proposes over what we have in tpm2-tss package, could we maybe make this PR a PR against tpm2-tss instead of a new aport?

Cogitri commented:
Oh, possibly, didn't realize it's the same package but more split up, sorry about that.

whooo commented:
Created a new PR, https://github.com/alpinelinux/aports/pull/11909

asac commented:
@whooo why are you making a new PR that moves? instead of making a PR that patches the existing tpm2-tss package and abandon this one... /me confused.

whooo commented:
@asac I moved tpm2-tss and patched it afterwards in the new PR.
The reason I moved it was that (as I understand it) packages in community are not expected to be maintained for longer then six months and they aren't moved into main (which long term I'm looking for)

asac commented:
lets first get a clean patch against the tpm2-tss package in without moving. @Cogitri moved the package to community for a reason as it was needed there, so lets not jump the gun and do too many things in one step. Also this patch still renames the package etc. can you please keep tpm2-tss as the main package name?

whooo commented:
I don't really understand why it's needed it community, can't packages in community depend on packages in testing?

Cogitri commented:
No

Cogitri commented on community/tpm2-tss/APKBUILD:
```diff
@@ -1,27 +1,86 @@
+$pkgname-static $pkgname-dev"
 source="https://github.com/tpm2-software/tpm2-tss/releases/download/$pkgver/tpm2-tss-$pkgver.tar.gz"
-options="!check"
+builddir="$srcdir/tpm2-tss-$pkgver"
```
No need to add this.

Cogitri commented on community/tpm2-tss/APKBUILD:
```diff
@@ -1,27 +1,86 @@
 pkgrel=0
-pkgdesc="TPM2 tss library"
+pkgdesc="TPM 2.0 TSS"
+license="BSD-2-Clause"
```
Just keep the license field at its old place

Cogitri commented on community/tpm2-tss/APKBUILD:
```diff
@@ -1,27 +1,86 @@
-pkgname="tpm2-tss"
-pkgver="2.3.1"
+# Contributor: Erik Larsson <who+alpine@cnackers.org>
+# Contributor: Alexander Sack <asac@pantacor.com>
```
Please just have one Maintainer and have it at the last line of this comment block (so the `Contributor` lines are above it).
